MoviesProject/CategoriesList/CategoriesListState.swift

```diff
@@ -3,13 +3,37 @@ import ComposableArchitecture
 public extension CategoriesList {
     @ObservableState
     struct State: Equatable {
-        public init() {}
+        
+        enum MovieCategories: Equatable {
```

Comment:
we usually use singular noun for enum naming  
```suggestion
        enum MovieCategory: Equatable {
```